R/kernel.r

```diff
@@ -312,6 +316,8 @@ initialize = function(connection_file) {
 
     executor <<- Executor$new(send_response = .self$send_response,
         abort_queued_messages = .self$abort_queued_messages)
+    comm_manager <<- Comm_Manager$new(send_response = .self$send_response)
+    assign('kernel', .self, envir = .GlobalEnv)
```

Comment:
No... but we welcome other ideas. We needed to have access to the kernel and comm_manager within the interpreter. Or first stab at it was to add the kernel to the environment.